cmds/zui/prov.go

```diff
@@ -43,13 +40,44 @@ func provisionRender(client zbus.Client, grid *ui.Grid, render *Flag) error {
 	go func() {
 		for counter := range counters {
 			rows := prov.Rows
-			rows[0][1] = fmt.Sprint(counter.Container)
-			rows[1][1] = fmt.Sprint(counter.Volume)
+			rows[1][1] = fmt.Sprint(counter.Container)
+			rows[1][3] = fmt.Sprint(counter.Volume)
 			rows[2][1] = fmt.Sprint(counter.Network)
-			rows[3][1] = fmt.Sprint(counter.VM)
-			rows[4][1] = fmt.Sprint(counter.ZDB)
-			rows[5][1] = fmt.Sprint(counter.Debug)
+			rows[2][3] = fmt.Sprint(counter.VM)
+			rows[3][1] = fmt.Sprint(counter.ZDB)
+			rows[3][3] = fmt.Sprint(counter.Debug)
+
+			render.Signal()
+		}
+	}()
+
+	sysMonitor := stubs.NewSystemMonitorStub(client)
+	stream, err := sysMonitor.CPU(context.Background())
+	if err != nil {
+		return errors.Wrap(err, "failed to start cpu monitor stream")
+	}
+
+	go func() {
+		for point := range stream {
+			prov.Mutex.Lock()
+			for _, cpu := range point {
```

Comment:
Each point returned from the stream has CPU stats for each CPU core on the node. so what u do now here is that u go over each cpu update the cell value and then render, which will basically show only LAST cpu usage not all of the them. 
What we need to do here is that the stream can return aggregated values (total cpu usage) + a list of individual cpus. then in ZUI can show the aggregated cpu load instead